sails/api/hooks/custom/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * custom hook
+ *
+ * @description :: A hook definition.  Extends Sails by adding shadow routes, implicit actions, and/or initialization logic.
+ * @docs        :: https://sailsjs.com/docs/concepts/extending-sails/hooks
+ */
+
+module.exports = function defineCustomHook(sails) {
+  return {
+    /**
+       * Runs when a Sails app loads/lifts.
+       */
+    initialize: async function () {
+
+      sails.log.info('Initializing hook... (`api/hooks/custom`)');
+
+      // TODO Mail service etc. will be innitiated here
+
```

Comment:
Empty lines and TODO has a spelling error (maybe add some details for initialize function to the function comment /* */)